lib/linguist/languages.yml

```
@@ -7607,6 +7607,14 @@ UnrealScript:
   codemirror_mode: clike
   codemirror_mime_type: text/x-java
   language_id: 382
+URScript:
+  type: programming
+  color: "#56A2D6"
+  extensions:
+  - ".script"
+  tm_scope: source.python
```

This should be the scope from the grammar:

```suggestion
  tm_scope: source.urscript
```